plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -143,7 +143,7 @@ data class Yarn2Config(
 )
 
 /**
- * The [Yarn 2+](https://v2.yarnpkg.com/) package manager for JavaScript.
+ * The [Yarn 2+ package manager](https://v2.yarnpkg.com/).
```

Comment:
What about PNPM, which currently says `The [fast, disk space efficient package manager](https://pnpm.io/).`?

Reply:
"Fast, disk space efficient package manager" comes from https://pnpm.io/.
What do you propose to do about it?

Reply:
Well, I though the point of *aligning* the docs is to make them all use the same working. So I was expecting

> The [PNPM package manager](https://pnpm.io/).